### PR TITLE
Docs - add to Run Report Guide page

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-scrape/issues/198

### CODE REVIEW

**Comment by odscjames** [comment]
[Scrape: document how to interpret scrapy summary stats](https://trello.com/c/Rsn2HF6v/115-scrape-document-how-to-interpret-scrapy-summary-stats)

File: docs/run-report-guide.rst [hunk 1]
```diff
@@ -24,3 +24,9 @@ For example, some of the Paraguay scrapers need an authentication token. The ser
 
 This means the presence of a non-200 line is not always a error, but it should always be checked.
 
+Asking for help to interpret problems
+-------------------------------------
+
+Unfortunately, it's hard to give clear guides on how to interpret problems as the advise can differ a lot for different scrapers.
```
Comment (pindec) [inline docs/run-report-guide.rst]: advise -> advice

File: docs/run-report-guide.rst [hunk 2]
```diff
@@ -24,3 +24,9 @@ For example, some of the Paraguay scrapers need an authentication token. The ser
...
+
+Please ask a developer for help with any issues and we will try and build up this documentation as common patterns are uncovered.
```
Comment (pindec) [inline docs/run-report-guide.rst]: Should we add 'For hosted Kingfisher, please ask ...' ?
Should we guide external people to add issues in kingfisher-scrape issues?

**Comment by odscjames** [comment]
Thank you @pindec ! Updated and ready for review.

**Pull request merged**